src/lit-html.ts

```diff
@@ -109,13 +109,11 @@ export function render(
  * An expression marker with embedded unique key to avoid
  * https://github.com/PolymerLabs/lit-html/issues/62
  */
-const attributeMarker = `{{lit-${Math.random()}}}`;
-
-const textMarkerContent = '_-lit-html-_';
-const textMarker = `<!--${textMarkerContent}-->`;
-const attrOrTextRegex = new RegExp(`${attributeMarker}|${textMarker}`);
-const lastAttributeNameRegex =
-    /((?:\w|[.\-_$])+)=(?:[^"']*|(?:["][^"]*)|(?:['][^']*))$/;
+const marker = `{{lit-${String(Math.random()).slice(2)}}}`;
```

Comment:
Removes the `0.` (particularly that "."), which is a special character in regexes. They don't really gain us any uniqueness, and just makes the regex work harder.